.github/workflows/test-android-e2e.yml

```diff
@@ -45,9 +47,12 @@ jobs:
           disable-animations: true
           profile: Nexus 6
           script: |
-            adb emu geo fix 72.58727 23.02579
+            adb emu geo fix 72.5228 23.0127
             ./gradlew app:connectedDebugAndroidTest -i -Pandroid.testInstrumentationRunnerArguments.class=com.aws.amazonlocation.ui.DefaultConnectionFlowSuite
```

Comment:
I do not see e2e tests workflow triggered in PR. Are we expecting e2e test only run after this pr is merged?